### ISSUE TITLE
RPCS3 ignoring and running opposite of main display settings.

### ISSUE THREAD

**Issue by BirdbrainN7** [RPCS3/rpcs3#14405]
### Quick summary

RPCS3 is ignoring and actively going opposite of Windows 11 "Main Display" settings. It won't open on my main display even if closed on that display and if I move it over to that display, the games, dropdown menus (config, file etc) still open on my secondary display, fullscreening anything also forces it to the second display.

I have tried restarting my PC, reinstalling the emulator itself and messing with my display settings to no luck. The only other note is that the emulator will swap to my "main" display if I set that display to be my secondary one. But that also means every other app I use will display on the wrong screen instead.

### Details

After updating to the latest build of RPCS3, the emulator has started exclusively displaying and starting on my secondary monitor. This includes dropdown menus (to the point they're completely detached from the main window), any new windows that open from the emulator (cheat menu, game render etc) and fullscreen will all go to the secondary monitor.

I cannot find any option within the emulator's settings to change this. Reinstalling the emulator, restarting my PC and messing with my PCs display settings has not changed anything.

It was working fine this morning (About 12 hours ago for me) and only started this behaviour after updating, RPCS3 is the only application experiencing this which leads me to believe it's a bug on the emulator's end.

### Attach a log file

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/12222775/RPCS3.log)

### Attach capture files for visual issues

[rpcs3monitorissue.zip](https://github.com/RPCS3/rpcs3/files/12222824/rpcs3monitorissue.zip)

### System configuration

Windows 11
GPU: GeForce 1660 ti
GPU Driver: GeForce Game Ready Driver 536.67
CPU: Intel(R) Core i7-10700K

**Comment by AniLeo** [comment]
Apparently this is an upstream Qt bug that was fixed on Qt 6.5.1

**Comment by BirdbrainN7** [comment]
> Apparently this is an upstream Qt bug ~that was fixed on Qt 6.5.1~

I'm guessing I'm going to have to wait for the bug to be fixed on QT and then for an update to the emulator with the fix? Or is there a way for me to solve this on my end?

Also sorry if I labelled this issue incorrectly before, first time using GitHub's issue system.

**Comment by MilkManEX** [comment]
Same issue here. Updated RPCS3 yesterday and it started doing this.

**Comment by dsdamod** [comment]
I also have this issue and I don't know if it's similar to anyone else but I found it's somehow connected to having a custom monitor resolution profile for both my monitors. I use CRU (custom resolution utility) to overclock my monitors and noticed that when I reset my monitor profiles to default this issue goes away. I also found that if I set a custom profile for only one monitor this issue goes away. It's only when i customize the profile of both my monitors that the RPCS3 window gets stuck on my secondary monitor.
Hope this bug will be resolved, for now i downgraded to v0.0.28-15417 as it is the last version before the Qt update.
